### PR TITLE
Restructure unit/e2e tests directory strucures 

### LINKED ISSUE #89: restructure unit/e2e tests directory strucures
align it to the app dir structure (by structure or by naming convention)

### PR DESCRIPTION
**Pull request by wcandillon**
fixes #89 

As described from the angularjs team at https://docs.google.com/document/d/1XXMvReO8-Awi1EZXAXS4PzDzdNvV6pGcuaF4Q9821Es/pub

### CODE REVIEW

File: tests/e2e/demo-scenario.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    it('Should display the fact table', function() {
+        report.filters.visitPage();
+        browser.waitForAngular();
+        report.factTable.visitPage();
+        expect(report.factTable.lineCount()).toBeGreaterThan(0);
+    });
+
+    it('Should display the preview', function() {
+        report.preview.visitPage();
+        //browser.waitForAngular();
```
Comment (MatthiasBrantner) [inline tests/e2e/demo-scenario.js]: ???

**Comment by wcandillon** [comment]
@MatthiasBrantner this is ready for review.

File: tests/e2e/demo-scenario.js (hunk 1 above)
Comment (wcandillon) [inline tests/e2e/demo-scenario.js]: this is addressed at https://github.com/28msec/nolap-report-editor/pull/114/files#diff-9e64145b95c95099aa0bd0b659aa2bd6R13

**Comment by wcandillon** [comment]
@dknochen: @MatthiasBrantner approved it. Could you have a look and merge it?

**Comment by dknochen** [comment]
@wcandillon super cool!

**Pull request merged**